src/ontology/mondo-edit.obo

```diff
@@ -133834,15 +133835,15 @@ xref: GARD:17843 {source="Orphanet:468666"}
 xref: OMIM:106190 {source="Orphanet:468666", source="MONDO:equivalentTo", source="DOID:0060603"}
 xref: Orphanet:468666 {source="MONDO:equivalentTo"}
 xref: UMLS:C5568836 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-is_a: MONDO:0005093 {source="MONDO:0018798-obsoleted"} ! skin disorder
 is_a: MONDO:0006527 {source="DOID:0060603", source="MONDO:Entailed", source="MONDO:Redundant"} ! anhidrosis
 is_a: MONDO:0019296 {source="Orphanet:468666"} ! subcutaneous tissue disorder
 is_a: MONDO:0021154 {source="Orphanet:468666"} ! dermis disorder
 intersection_of: MONDO:0006527 ! anhidrosis
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6181 ! ITPR2
+relationship: excluded_subClassOf MONDO:0005093 {source="MONDO:0018798-obsoleted", source="https://orcid.org/0000-0001-5208-3432"} ! skin disorder
```

Comment:
another parent is "dermis disorder", which is a "skin disorder". So excluded this SC does not make sense. 

Also this term is a "anhidrosis" (= sweat gland disorder) "with normal sweat gland". 

This term requires more review, but at minimum, we should either not exclude SC "skin disorder", or also exclude SC "dermis disorder"